### PR TITLE
[Paddle FE] enble paddle.roll

### PR DESCRIPTION
**Pull request by openvino-dev-samples**
### Tickets:
 - *CVS-150687*

### CODE REVIEW

**Comment by yuxu42** [comment]
Hi @meiyang-intel  Could you please take a review? Thanks!

File: src/frontends/paddle/src/op/roll.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs roll(const NodeContext& node) {
+    auto input_node = node.get_input("X");
+    Output<Node> shifts_node;
+    if (node.has_input("ShiftsTensor")) {
```
Comment (meiyang-intel) [inline src/frontends/paddle/src/op/roll.cpp:15]: Can you also add a testcase with "ShiftsTensor" input?

Comment (openvino-dev-samples) [reply src/frontends/paddle/src/op/roll.cpp:15]: hi @meiyang-intel I think this testcase will be triggered when shift is variable as `roll_test_2`:
https://github.com/PaddlePaddle/Paddle/blob/release/2.6/python/paddle/tensor/manipulation.py#L1890

Comment (meiyang-intel) [reply src/frontends/paddle/src/op/roll.cpp:15]: The code show that it will be triggered when shifts is variable. But in this testcase shifts=1

**Review by meiyang-intel — APPROVED** [review]

**Review by ceciliapeng2011 — APPROVED** [review]

**Comment by openvino-dev-samples** [comment]
Hi @ceciliapeng2011 @meiyang-intel can we merge it ?

**Comment by yuxu42** [comment]
it looks like some tests are not complete @openvino-dev-samples @ceciliapeng2011

**Pull request merged**